src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -30,7 +30,7 @@ function ProductDetails({ product: staleProduct }: Props) {
       name: variantName,
       brand: { name: brandName },
       isVariantOf: { name, productGroupID: productId },
-      image: [img],
+      image: productImages,
```

Comment:
When changing this, my initial thought was that it would be "one less thing to do" when we integrate the gallery to the PDP. Just pass the "productImages" to the component and it's done.